src/bootstrap.php

```diff
@@ -779,18 +779,26 @@ function (Accept $type, string $number) use ($app) {
         if (false === isset($app['articles'][$number])) {
             throw new NotFoundHttpException('Article not found');
         }
-
         $article = $app['articles'][$number];
-
+        $headers = ['Content-Type' => $type->getNormalizedValue()];
         $content = $article['relatedArticles'] ?? [];
 
+        if ($type->getParameter('version') < 2) {
+            $headers['Warning'] = sprintf('299 elifesciences.org "Deprecation: Support for version %d will be removed"', $type->getParameter('version'));
```

Comment:
Can't introduce this one this time. As we haven't marked it as deprecated in the schema and we should leave that decision to @lsh-0.